src/shared/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import Tooltip, { TooltipProps } from './Tooltip'
+import { Text, Button, ChannelPreview } from '@/shared/components'
+import { Meta, Story } from '@storybook/react'
+
+export default {
+  title: 'Shared/Tooltip',
+  component: Tooltip,
+  argTypes: {
+    text: { control: 'text', defaultValue: 'Lorem ipsum dolor sit amet!' },
+    arrowDisabled: { control: 'boolean', defaultValue: false },
```

Comment:
```suggestion
    text: { defaultValue: 'Lorem ipsum dolor sit amet!' },
    arrowDisabled: { defaultValue: false },
```
I checked it. You don't really need that - storybook handles it

Reply:
Good to know! 👍